### PR TITLE
Updated stable tags

### PR DESCRIPTION
**Pull request by jernejpregelj**
- Update stable tags
- **Still needs update to pleuston tag**

### CODE REVIEW

**Comment by jernejpregelj** [comment]
Do not merge! Still needs update on Pleuston tag

**Comment by H34D** [comment]
> Do not merge! Still needs update on Pleuston tag

i just tried with `pleuston` `v0.4.1` and it works quite well?

**Comment by jcortejoso** [comment]
@jernejpregelj pleuston `v0.4.1` works for you?

File: start_ocean.sh [hunk 1]
```diff
@@ -13,9 +13,9 @@ DIR="${DIR/ /\\ }"
 COMPOSE_DIR="${DIR}/compose-files"
 
 # Default versions of Aquarius, Brizo, Keeper Contracts and Pleuston
-export AQUARIUS_VERSION=${AQUARIUS_VERSION:-v0.2.9}
-export BRIZO_VERSION=${BRIZO_VERSION:-v0.3.10}
-export KEEPER_VERSION=${KEEPER_VERSION:-v0.9.7}
+export AQUARIUS_VERSION=${AQUARIUS_VERSION:-v0.3.4}
+export BRIZO_VERSION=${BRIZO_VERSION:-v0.3.12}
+export KEEPER_VERSION=${KEEPER_VERSION:-v0.10.3}
 export PLEUSTON_VERSION=${PLEUSTON_VERSION:-v0.4.0}
```
Comment (jcortejoso) [inline start_ocean.sh]: export PLEUSTON_VERSION=${PLEUSTON_VERSION:-v0.4.1}

**Comment by jernejpregelj** [comment]
Not everything works with 0.4.1, Pleuston needs 0.4.2 release that includes squid-js update.

**Comment by jernejpregelj** [comment]
Tested and works for me. @ssallam Please confirm from your side so this can be merged.

**Pull request merged**